### PR TITLE
micro refactoring of WEBrick modules

### PR DESCRIPTION
**Pull request by bohdan-malets**
a proposal of small for tabulation, initialization and readability changes of WEBrick modules

### CODE REVIEW

File: lib/webrick/httpauth/authenticator.rb [hunk 1]
```diff
@@ -27,19 +27,10 @@ module Authenticator
...
-      attr_reader :userdb
-
-      ##
-      # The logger for this authenticator
-
-      attr_reader :logger
+      # realm  - the realm this authenticator covers
+      # userdb - the user database for this authenticator
+      # logger - the logger for this authenticator
+      attr_reader :realm, :userdb, :logger
```
Comment (windwiny) [inline lib/webrick/httpauth/authenticator.rb]: `rdoc` can not  recognize attribute's comment.

Comment (bohdan-malets) [inline lib/webrick/httpauth/authenticator.rb]: @windwiny, how should I write attribute's comment for rdoc correctly?

Comment (windwiny) [inline lib/webrick/httpauth/authenticator.rb]: Just like original text.
You can write a simple ruby script, and use `rdoc xx.rb` generate doc and check `doc/index.html` context.

Comment (bohdan-malets) [inline lib/webrick/httpauth/authenticator.rb]: @windwiny, thanks! don't hesitate to push something into my branch ;)

File: lib/webrick/cgi.rb [hunk 2]
```diff
@@ -105,14 +97,14 @@ def res.status_line
...
+        req.script_name    = (env["SCRIPT_NAME"] || File.expand_path($0)).dup
+        req.path_info      = (env["PATH_INFO"] || "").dup
+        req.query_string   = env["QUERY_STRING"]
+        req.user           = env["REMOTE_USER"]
         res.request_method = req.request_method
-        res.request_uri = req.request_uri
+        res.request_uri    = req.request_uri
+        res.keep_alive     = req.keep_alive?
         res.request_http_version = req.http_version
-        res.keep_alive = req.keep_alive?
```
Comment (nobu) [inline lib/webrick/cgi.rb]: Why does `keep_alive` need to come before `request_http_version`?

Comment (bohdan-malets) [inline lib/webrick/cgi.rb]: @nobu, as for me
```
obj.attr1 = 1
obj.attr2 = 2
obj.attr4 = 4
obj.attribute3 = 3
```
is more readable than:
```
obj.attr1 = 1
obj.attr2 = 2
obj.attribute3 = 3
obj.attr4 = 4
```
am I correct?

Comment (nobu) [inline lib/webrick/cgi.rb]: You mean attribute name lengths?
I see it's your preference.

Comment (bohdan-malets) [inline lib/webrick/cgi.rb]: @nobu, `keep_alive ` is moved back back in place.

**Comment by hsbt** [comment]
It's not +micro+ refactoring :innocent: 

Please separate `syntax changes`, `condition changes` and more each pull requests.

**Pull request closed without merging**